geometry/Overview.bs

```diff
@@ -436,9 +436,7 @@ In the following sections, terms have the following meaning:
 </dl>
 
 <pre class='idl'>
-[Constructor,
- Constructor(DOMString transformList),
- Constructor(sequence&lt;unrestricted double> numberSequence),
+[Constructor(optional (DOMString or sequence&lt;unrestricted double>) init = [1, 0, 0, 1, 0, 0]),
```

Comment:
You can only use [] as a literal, not an actual array. You'll have to just omit the default value and define it in prose.